en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -17,115 +17,114 @@ review-ticket: https://github.com/programminghistorian/ph-submissions/issues/374
 topics: [r]
 difficulty: 3
 activity: analyzing
-abstract: In this lesson, you will learn how to download YouTube comments and use the R programming language to analyze the political valences of a corpus with the Wordfish algorithm.
+abstract: In this lesson, you will learn how to download YouTube comments and use the R programming language to analyze the political values of a corpus with the Wordfish algorithm.
 avatar_alt: Visual description of lesson image
 doi: XX.XXXXX/phen0000
 ---
 
 {% include toc.html %}
 
-# Introduction
-YouTube is the most popular web-based video sharing platform in the world, with billions of users viewing and uploading videos each month. In this lesson, you will learn how to download YouTube comments and analyze the text data using the natural language processing algorithm, Wordfish.
+## Introduction
 
-This lesson will walk introduce readers to the way YouTube videos, and the comments users write in response to videos, can be a valuable resource for researchers interested in discourse analysis. Readers will learn how to use the R programming language and the text mining algorithm, Wordfish, to explore a collection of YouTube comments. Wordfish is useful for measuring political valence in a set of documents, and for this lesson's sample dataset, we've collected comments submitted by viewers of Black Lives Matter videos posted to YouTube by right- and left-leaning news sources in the United States in 2020. 
+YouTube is the most popular web-based video sharing platform in the world, with billions of users viewing and uploading videos to it each month. In this lesson, you will learn how to download YouTube comments and analyze the text data using the natural language processing algorithm Wordfish, an [unsupervised text-scaling algorithm](link-to-definition) used for interpreting the primary dimensions of latent meaning within bodies of text data.
 
-This lesson will guide you through three key steps for 1) data collection, 2) cleaning and modeling, and  3) analysis and visualization. 
+This lesson will introduce readers to the way both YouTube videos and the comments written by users in response can be a valuable resource for researchers who are interested in discourse analysis. Readers will learn how to use the R programming language and the text mining algorithm Wordfish to explore a collection of YouTube comments. Wordfish is useful for measuring political valence in a set of documents, and for this lesson's sample dataset, we've collected comments submitted by viewers of Black Lives Matter videos posted to YouTube by right- and left-leaning news sources in the United States in 2020. 

```

Comment:
Can we add a link for 'Black Lives Matter', to help give context to our readers who may not be as up-to-date on American politics?

Perhaps [the BLM wikipedia page](https://en.wikipedia.org/wiki/Black_Lives_Matter), or the [BLM website about page](https://blacklivesmatter.com/about/) itself?

Reply:
Yeah the wikipedia page works